traitcuration/traits/datasources/dummy.py

```diff
@@ -14,14 +14,14 @@ def import_dummy_data():
     Mapping.objects.all().delete()
     Trait.objects.all().delete()
     OntologyTerm.objects.all().delete()
-    User.objects.all().delete()
+    User.objects.exclude(email="admin@admin.com").delete()
```

Comment:
Does this mean that we have admin@admin.com hardcoded somewhere as the email for the administrative user?

I feel like it would be better to either replace it with something sensible (e.g. eva-dev@ebi.ac.uk), or at least use the example.com domain instead of admin.com (which is a domain which actually exists)

Reply:
This is just for development purposes so that we can use the dummy admin credentials that are pinned in the Slack channel. 

I am manually adding admin@admin.com each deployment but I want to add a bash script that is executed upon deployment release and adds that user automatically.

I don't think the domain not existing is a problem, the sole purpose of those creds is to access the admin site, but if you think it would be better, I could change it to something like 'admin@example.com'

Reply:
> I don't think the domain not existing is a problem

No, my concern is the opposite one, that the admin.com domain _does_ exist and hosts some real life webiste. IANA [recommends](https://www.iana.org/domains/reserved) that all such placeholder emails/addresses use the example.com domain, which is specifically reserved for this purpose. I agree this is not critical, just a good practice to follow

Reply:
OK, I'm going to resolve this conversation for now; but I've added a point to #58 to get rid of this in production